### DISCUSSION TITLE
能否优化竖屏视频的播放器？

### DISCUSSION THREAD

**Discussion by userzzzq** [the1812/Bilibili-Evolved#4665] [功能建议 / Ideas]
播放竖屏视频时，仍然是16:9的播放器，造成空间浪费，以及视频显示得太小，能否改成像youtube的播放器那样，竖屏视频默认撑满网页？

对比图：

不知道怎么回事，上传图片总是失败，外链图片：https://ibb.co/cvCbYnM

**Comment by the1812** [comment]
宽屏 / 网页全屏 / 全屏切一下不就大了

**Reply by userzzzq** [reply]
不一样的，网页全屏状态下不能看评论以及其它信息，而像youtube那样则不影响看评论。

**Reply by Geeyun-JY3** [reply]
能理解这需求，但说真的，这真应该是破站自己来做的事，破站网页端现在这个体验也不知道是对竖屏视频态度又改了还是单纯不重视网页端
PS：YouTube 这功能是不是还在灰度，去搜了下这个视频进去看好像和你附的图不太一样。
